### ISSUE TITLE
LIR Circle with Type=address doesnt respect the Circle area

### ISSUE THREAD

**Issue by vasile** [openTdataCH/ojp-sdk#17]
https://tools.odpch.ch/ojp-api-explorer-v2/?gist=98e22bc3de8d170dc9ec19e19df90c4e

<img height="400" alt="image" src="https://github.com/openTdataCH/ojp-sdk/assets/113994/0b1d2bbe-f41f-4ce7-bdff-6e62bc706695">

The filter seems to use a rectangle instead of circle

```
<GeoRestriction>
  <Circle>
    <Center>
      <siri:Longitude>7.416356</siri:Longitude>
      <siri:Latitude>46.926923</siri:Latitude>
    </Center>
    <Radius>200</Radius>
  </Circle>
</GeoRestriction>
```

Request
`/POST https://api.opentransportdata.swiss/ojp20`
XML: https://gist.github.com/vasile/98e22bc3de8d170dc9ec19e19df90c4e

**Comment by TO-mdv** [comment]
@vasile in the Request, you search for a radius of 200m -> diameter of 400m. All the results seems to be in the diameter 400m:
![image](https://github.com/openTdataCH/ojp-sdk/assets/72090402/538af598-9831-4715-86a8-e2a4f3cf59ad)

It seems like your visualizer (which by the way is verry cool!) shows only a radius of 100m, could that be?

**Comment by vasile** [comment]
```
It seems like your visualizer (which by the way is verry cool!) shows only a radius of 100m, could that be?
```
Hi @TO-mdv - it should be 200m,  if you compare with the scale control

<img width="315" alt="image" src="https://github.com/openTdataCH/ojp-sdk/assets/113994/f6837171-51eb-4bce-a6c8-d598683f8f1e">

**Comment by TO-mdv** [comment]
Ah yes, you are right. Maybe our Application swaps radius and diameter, I check that.

**Comment by TO-mdv** [comment]
Bug created: [SH-2893](https://youtrack.mentz.net/issue/SH-2893) OJP2.0 LIR liefert Ergebnisse ausserhalb Circle -> LIR-2e_Address_Circle

**Comment by vasile** [comment]
Thanks for the follow-up @TO-mdv - however please assign it with low-priority because `Rectangle` GeoRestriction is used for now.

**Comment by TO-mdv** [comment]
Fixed on INT, Release in PROD 25.7.2024


Some elements are still out of the Circle, because the street cut the circle somewhere at beginning:

![Image](https://github.com/openTdataCH/ojp-sdk/assets/72090402/70e06e55-175a-4316-81e2-51475df940a6)
